docs/when.rst

```diff
@@ -30,61 +21,36 @@ Merging the two task lists is left to the user.
 A better idea would've been to use ``git`` to synchronize the ``todo.txt``
 file, which is at least able to resolve some basic conflicts.
```

Comment:
Also worth mentioning the lack of clients for lots of devices/platforms, while *dav support is kinda universal.